src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -171,8 +171,10 @@
           </tr>
           </thead>
           <tbody>
-          <tr ng-repeat="event in ctrl.replicaSetEvents.events">
+          <tr ng-repeat="event in ctrl.events">
             <td class="kd-replicasetdetail-table-cell kd-replicasetdetail-table-message">
+              <i ng-if="event.type == 'Warning'"
```

Comment:
Dont insert logic into templates. Move all logic to controllers. E.g., you can create method: ctrl.isEventWarning(...)